### PR TITLE
Replace chat type checks with bot.chatType and ctx.hasChatType

### PR DESCRIPTION
**Pull request by KnorpelSenf**
Are there more cases like this?

### CODE REVIEW

**Comment by roj1512** [comment]
https://github.com/grammyjs/website/blob/main/site/docs/plugins/ratelimiter.md?plain=1#L203-L239

**Comment by KnorpelSenf** [comment]
Do you want to quickly fix it yourself?

**Review by roj1512 — APPROVED** [review]
LGTM.

File: notifier/main.ts [hunk 1]
```diff
@@ -42,6 +42,7 @@ bot.chatType(["group", "supergroup"]).filter((ctx) =>
     return;
   }
   branch = (await res.json())?.head?.ref;
+  ctx.hasChatType([''])
```
Comment (KnorpelSenf) [inline notifier/main.ts:45]: What

Comment (roj1512) [reply notifier/main.ts:45]: my bad

**Review by KnorpelSenf — COMMENTED** [review]
LGTM.

**Pull request merged**